kroxylicious-systemtests/pom.xml

```diff
@@ -186,6 +186,7 @@
                                 <include>**/ST*.java</include>
                                 <include>**/*ST.java</include>
                             </includes>
+                            <excludedGroups>unit</excludedGroups>
```

Comment:
Why does `ContainerTemplatesTest` get picked up at all? It doesn't match the `include` entries...

Reply:
it's the `-Dtests`, overriding the includes

> Specify this parameter to run individual tests by file name, overriding the parameter includes and excludes. Each pattern you specify here will be used to create an include pattern formatted like **/${test}.java, so you can just type -Dtest=MyTest to run a single test called "foo/MyTest.java". The test patterns prefixed with a ! will be excluded.